Flow.Launcher.Infrastructure/Alphabet.cs

```diff
@@ -19,160 +19,45 @@ public interface IAlphabet
 
     public class Alphabet : IAlphabet
     {
-        private readonly HanyuPinyinOutputFormat Format = new HanyuPinyinOutputFormat();
-        private ConcurrentDictionary<string, string[][]> PinyinCache;
-        private BinaryStorage<Dictionary<string, string[][]>> _pinyinStorage;
+        private ConcurrentDictionary<string, string> _pinyinCache;
         private Settings _settings;
-         
+
         public void Initialize([NotNull] Settings settings)
         {
             _settings = settings ?? throw new ArgumentNullException(nameof(settings));
-            InitializePinyinHelpers();
-        }
-
-        private void InitializePinyinHelpers()
-        {
-            Format.setToneType(HanyuPinyinToneType.WITHOUT_TONE);
-
-            Stopwatch.Normal("|Flow Launcher.Infrastructure.Alphabet.Initialize|Preload pinyin cache", () =>
-            {
-                _pinyinStorage = new BinaryStorage<Dictionary<string, string[][]>>("Pinyin");
-                
-                lock(_pinyinStorage)
-                {
-                    var loaded = _pinyinStorage.TryLoad(new Dictionary<string, string[][]>());
-
-                    PinyinCache = new ConcurrentDictionary<string, string[][]>(loaded);
-                }
-
-                // force pinyin library static constructor initialize
-                PinyinHelper.toHanyuPinyinStringArray('T', Format);
-            });
-            Log.Info($"|Flow Launcher.Infrastructure.Alphabet.Initialize|Number of preload pinyin combination<{PinyinCache.Count}>");
-        }
-
-        public string Translate(string str)
-        {
-            return ConvertChineseCharactersToPinyin(str);
-        }
```

Comment:
The Translate method, could we keep the `return ConvertChineseCharactersToPinyin(str);` so we know the code is in relation to pinyin conversion. And the actual code that does the conversion can stay in ConvertChineseCharactersToPinyin method.

In the future the Translate method can expand to facilitating other language conversions.

Reply:
I think since bao has implement the ialphabet interface, if in the future we want to add more translation, we should add a new alphabet class that contains another language translation.
Bao has removed that in wox but I think it can be kept for future expansion.

Reply:
Yeah i agree. We should really use inheritance/polymorphism here, but I dont know what languages we can translate. I guess my concern here is that Translate is not clear enough to indicate it's intended behaviour, although i do want to keep it generic naming so later on translate can take on different forms for different language translations. What do you think adding comments in here to explain what sort of translation it is right now? At least the user can get an idea since this is also a public method.

Reply:
I think we can change the name of the `Alphabet` that implement the `IAlphabet` to indicate that which alphabet it is. For example, for the alphabet that for pinyin, we may change it to pinyinAlphabet? Although the name is a little bit long hhh.
That shall tell users what the alphabet is doing.

Reply:
Sounds good!

Reply:
Is it possible to rename Alphabet to PinyinAlphabet?

Reply:
Sure